aspnetcore/signalr/get-started-signalr-core.md

```diff
@@ -19,60 +19,100 @@ This tutorial teaches the basics of building a real-time app using SignalR for A
 
-## Prerequisites
+# Prerequisites
 
 Install the following software:
 
+# [Visual Studio](#tab/visual-studio)
+
 * [.NET Core 2.1.0 Preview 1 SDK](https://www.microsoft.com/net/download/dotnet-core/sdk-2.1.300-preview1) or later
-* [Visual Studio 2017](https://www.visualstudio.com/downloads/) version 15.6 or later with the ASP.NET and web development workload
+* [Visual Studio 2017](https://www.visualstudio.com/downloads/) version 15.6 or later with the **ASP.NET and web development** workload
 * [npm](https://www.npmjs.com/get-npm)
 
+# [Visual Studio Code](#tab/visual-studio-code)
+
+* [.NET Core 2.1.0 Preview 1 SDK](https://www.microsoft.com/net/download/dotnet-core/sdk-2.1.300-preview1) or later
+* [Visual Studio Code](https://code.visualstudio.com/download) 
+* [C# for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=ms-vscode.csharp)
+* [npm](https://www.npmjs.com/get-npm)
+
+-----
+
 ## Create an ASP.NET Core project that hosts SignalR client and server
 
-1. Use the **File** > **New Project** menu option and choose **ASP.NET Core Web Application**. Name the project `SignalRChat`.
+# [Visual Studio](#tab/visual-studio)
+
+1. Use the **File** > **New Project** menu option and choose **ASP.NET Core Web Application**. Name the project *SignalRChat*.
 
   ![New Project dialog in Visual Studio](get-started-signalr-core/_static/signalr-new-project-dialog.png)
 
-2. Select **Web Application** to create a project using Razor Pages. Then select **Ok**. Be sure that **ASP.NET Core 2.1** is selected from the framework selector, though SignalR runs on older versions of .NET.
+2. Select **Web Application** to create a project using Razor Pages. Then select **OK**. Be sure that **ASP.NET Core 2.1** is selected from the framework selector, though SignalR runs on older versions of .NET.
 
   ![New Project dialog in Visual Studio](get-started-signalr-core/_static/signalr-new-project-choose-type.png)
 
-  The libraries that host SignalR server-side code are included in the project template. Install the client-side JavaScript separately with [npm](https://www.npmjs.com/).
+3. Right-click the project in **Solution Explorer** > **Add** > **New Item** > **npm Configuration File**. Name the file *package.json*.
+
+4. Run the following command in the **Package Manager Console** window, from the project root:
 
-  ```console
-   npm install @aspnet/signalr
-  ```
+    ```console
+      npm install @aspnet/signalr
+    ```
+5. Copy the *signalr.js* file from *node_modules\\@aspnet\signalr\dist\browser* to the *wwwroot\lib* folder in your project.
 
-3. Copy the *signalr.js* from *node_modules\\@aspnet\signalr\dist\browser* to the *wwwroot\lib* folder in your project.
+# [Visual Studio Code](#tab/visual-studio-code)
+
+From the **Integrated Terminal**, run the following command:
+ 
+```console
+dotnet new razor -o SignalRChat
+```
```

Comment:
You'll need to add the npm instructions in the VS Code tab too. Here are the 2 commands to run from the project root:
```console
npm init -y
npm install @aspnet/signalr
```

It's this first command that creates the *package.json* file.